game/script-farewells.rpy

```diff
@@ -2,6 +2,7 @@ default persistent._farewell_database = dict()
 
 default persistent.jn_player_first_farewell_response = None
 default persistent.jn_player_force_quit_state = 1
+default persistent.jn_player_restarted = False
```

Comment:
```suggestion
default persistent._jn_player_be_right_back = False
```

It isn't noted anywhere so it might not have been clear, but can we use a `_` prefix when declaring persistent variables? This is the convention Python uses for marking something as for internal use only - would be best to stick with it for anything new!

As per @RaionArt's feedback - a topic about restarting specifically might be a little on the nose: I think it'd be worth seeing how this works as a "Be right back!" style farewell/greeting!